src/java/org/apache/cassandra/service/StorageProxyMBean.java

```diff
@@ -103,7 +103,4 @@ public interface StorageProxyMBean
     boolean getCheckForDuplicateRowsDuringCompaction();
     void enableCheckForDuplicateRowsDuringCompaction();
     void disableCheckForDuplicateRowsDuringCompaction();
-
-    int getMaxPaxosBackoffMillis();
```

Comment:
@jtgrabowski do you know what is the use case for this interface and if those are safe to remove? Just wanted to avoid adding the min/max props in both StorageProxy && PaxosUtils - they'll look safe to remove but making sure.

They were introduced here: https://github.com/datastax/cassandra/pull/1444

Reply:
The use case is to be able to on-line tweak the sleep time. This is important, because it reduces the experimentation effort by orders of magnitude (no need to restart the nodes etc).
Instead of removing, please add similar methods for the min bound :)

Reply:
you can create a separate mbean for `PaxosUtils` and move the methods there

Reply:
Testing locally:
```
% jmx_get -m org.apache.cassandra.service:type=Paxos MaxPaxosBackoffMillis
50
% jmx_get -m org.apache.cassandra.service:type=Paxos MinPaxosBackoffMillis
5
% jmx_set -m org.apache.cassandra.service:type=Paxos MaxPaxosBackoffMillis 60

% jmx_set -m org.apache.cassandra.service:type=Paxos MinPaxosBackoffMillis 6

% jmx_get -m org.apache.cassandra.service:type=Paxos MaxPaxosBackoffMillis
60
% jmx_get -m org.apache.cassandra.service:type=Paxos MinPaxosBackoffMillis
6
```